examples/deno-deploy/README.md

```diff
@@ -0,0 +1,6 @@
+- [Deno Deploy](https://deno.com/deploy)
+- Vanilla TRPCClient in node
+
+---
+
+Created by [tomlienard](https://github.com/tomlienard).
```

Comment:
Please add something on how this is run

Ideally also add it to the CI to make sure we don't break it :)

Reply:
That's why it kept the `package.json` before, because the CI uses turbo to run the `dev-test` scripts. Should I add it back?

Reply:
Nonono, I mean it should run with deno, not node